tests/integrations/test_dsm.py

```diff
@@ -72,16 +74,7 @@ def test_dsm_rabbitmq(self):
             tags=("direction:in", "topic:systemTestRabbitmqQueue", "type:rabbitmq"),
         )
 
-
-@released(cpp="?", golang="?", nodejs="?", php="?", python="?", ruby="?", java="?")
-@released(dotnet="2.29.0")
-@scenarios.integrations
-class Test_DsmRabbitmq_Dotnet:
-    """ Verify DSM stats points for RabbitMQ, specifically for Dotnet """
-
-    def setup_dsm_rabbitmq(self):
-        self.r = weblog.get("/dsm?integration=rabbitmq")
-
+    @irrelevant(context.library != "dotnet", reason="legacy dotnet behavior")
     def test_dsm_rabbitmq(self):
```

Comment:
you have two `test_dsm_rabbitmq` test methods :) 

You can name the second one `test_dsm_rabbitmq_dotnet_legacy`.

Reply:
good catch! fixed, thanks!